ts/input/tex.ts

```diff
@@ -173,6 +173,7 @@ export class TeX<N, T, D> extends AbstractInputJax<N, T, D> {
    * @override
    */
   public compile(math: MathItem<N, T, D>, document: MathDocument<N, T, D>): MmlNode {
+xxx();
```

Comment:
That should probably be removed.

Reply:
OOPS!  That was my testing code to force a compile error.

Reply:
Done.